test-case/check-pause-suspend-resume-release.sh

```diff
@@ -146,7 +145,7 @@ expect {
         }
 
         #enter suspend-resume cycle once per pause instance
-        set retval [catch { exec bash check-suspend-resume.sh -l 1 } msg]
+        set retval [catch { exec bash /home/ubuntu/sof-test/test-case/check-suspend-resume.sh -l 1 } msg]
```

Comment:
Instead of hard coded path, can we use BASH_SOURCE[0] in expect?

Reply:
Agreed, `/home/ubuntu/` is not going to work for most people. It worked with a relative path before, what changed?

Reply:
I will change the code and use the BASH_SOURCE[0] to get the dirname.